scripts/autogen.py

```diff
@@ -511,6 +511,8 @@ def make_md_source_for_entry(self, entry, path_stack, title_stack):
                     "missing {{toc}} tag." % (template_path,)
                 )
             template = template.replace("{{toc}}", toc)
+        if "keras_nlp/" in path_stack:
```

Comment:
It's always going to be at a specific position right? You can refer to it by index